src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static final User USER_DEFAULT_1 = new User(1L,"login1", "123456", 50);
+    private static final User USER_DEFAULT_2 = new User(2L, "login2", "789012", 25);
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        StorageDao storageDao = new StorageDaoImpl();
+        storageDao.add(USER_DEFAULT_1);
+        storageDao.add(USER_DEFAULT_2);
```

Comment:
do you need to add people to storage before all tests?

let's better do this in separate tests where we need it
and clear storage after each test